js/levels.js

```diff
@@ -19,4 +19,7 @@ function createTestLevel()
     [door.none, door.top, door.both, door.both, door.both, door.top],
```

Comment:
could you replace one of the door.both with door.chessMate and one with door.chessStale so they are in the current labyrinth?

Reply:
Sure